README.md

```diff
@@ -0,0 +1,51 @@
+# TempleApp
+
+This app is basically a cost-efficient way of maintaining records of poojas, donations made by people and also all the financial records in temples. Using this app, people in the temple counter will be able to enter all the details regarding the poojas they want or some donations they want to do for the temple. This app is connected to a printer so that it will print the details of pooja registrations of users.
+The app has a login interface and only some people like General secretary, president and treasurer of the temple can log in and be able to access the complete database.
+## Getting Started
+
+These instructions will get you a copy of the project up and be running on your local machine for development and testing purposes.
+### Prerequisites
+[Android Studio](https://developer.android.com/studio), with a recent version of the Android SDK.
+
+### Setting up your development environment
+
+- Download and install Git and add it to your PATH
+
+- Fork the templeApp project from``` https://github.com/amfoss/TempleApp```
+
+- Clone your fork of the project locally. At the command line:
+
+```git clone https://github.com/YOUR-GITHUB-USERNAME/templeApp```
+
+If you prefer not to use the command line, you can use Android Studio to create a new project from version control using ```https://github.com/YOUR-GITHUB-USERNAME/templeApp```
+
+Open the project in the folder of your clone from Android Studio. To run the project, click on the green arrow at the top of the screen
```

Comment:
`First, build the project. If there are any missing dependencies install them by clicking on the links provided by Android Studio itself. Once the project is successfully built, run it using the green arrow......`

Make proper sentences of the thing above too and write it.